### PR TITLE
chapter4 :Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by hieunguyen061197**
ready

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-module ApplicationHelper
+odule ApplicationHelper
```
Comment (thanhhungchu95) [inline app/helpers/application_helper.rb]: module

**Comment by hieunguyen061197** [comment]
ready

**Review by thanhhungchu95 — COMMENTED** [review]
(merged)

**Comment by hieunguyen061197** [comment]
image

**Pull request merged**